src/my-app/src/SingleProductPageContent.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react'
+import Header from './components/Header'
+import Footer from './components/Footer'
+import products from './SingleProductPageMatter'
+let quantity = 1
+function SingleProductPageContent () {
```

Comment:
pass single "product" as a parameter that would be called while clicking on a card from the list of products.